frontend/src/metabase/components/TokenField.jsx

```diff
@@ -33,6 +33,7 @@ export type LayoutRendererProps = {
   optionsList: ?React.Element,
   isFocused: boolean,
   isAllSelected: boolean,
+  isFiltered: boolean,
```

Comment:
Note: this is already being passed, it just wasn't in the type definition.

https://github.com/metabase/metabase/blob/776dba2a49ab6e34d6ce39c0808a784baf1ae249/frontend/src/metabase/components/TokenField.jsx#L638-L645